Remotion/Web/Development.WebTesting/HostingStrategies/DockerHosting/DockerCommandLineClient.cs

```diff
@@ -54,19 +54,22 @@ public void Pull (string imageName)
     public string Run (
         IDictionary<int, int> ports,
         IDictionary<string, string> mounts,
+        IDictionary<string, string> environmentVariables,
         string imageName,
         string? isolationMode,
         string? hostname,
         bool remove,
         string? entryPoint,
+        string? workingDirectory,
         string? args)
     {
       ArgumentUtility.CheckNotNull("ports", ports);
       ArgumentUtility.CheckNotNull("mounts", mounts);
+      ArgumentUtility.CheckNotNull("environmentVariables", environmentVariables);
       ArgumentUtility.CheckNotNullOrEmpty("imageName", imageName);
       ArgumentUtility.CheckNotEmpty("hostname", hostname);
       ArgumentUtility.CheckNotEmpty("entryPoint", entryPoint);
-      ArgumentUtility.CheckNotEmpty("args", args);
+      ArgumentUtility.CheckNotEmpty("workingDirectory", workingDirectory);
```

Comment:
```suggestion
      ArgumentUtility.CheckNotEmpty("workingDirectory", workingDirectory);
      ArgumentUtility.CheckNotEmpty("args", args);
```

Reply:
The idea was that a string typically is null to indicate not-there and empty is forbidden unless it's a valid value.

Down in the methid, the checks for not-null vs string.isnullorempty - that's a complete discussion and there's no hard and fast rule. probably, most of the time you just check for string.isnullorempty because it frees up the code to never run into a bug there. since the code already uses the not-null check instead, swithching this now seems odd.